_source/api/logzio-public-api.yml

```diff
@@ -2195,6 +2314,216 @@ definitions:
+    properties:
+      pageNumber:
+        type: integer
+        format: int32
+        description: If you overshoot the page number, it will return empty with no results, but it won't fail the request.
+        default: 1
+        example: 2
+      pageSize:
+        type: integer
+        format: int32
+        description: Controls the number of results per page. Valid inputs are 1 to 1000.
+        maximum: 1000
+        default: 25
+        example: 100
+  PagedSearchResponseMapStringObject:
+    type: object
+    properties:
+      total:
+        description: Returns the total number of logs linked to the security event specified in the query. This number is fixed and not affected by pagination.
+        type: integer
+        format: int32
+        example: 5
+      results:
+        type: array
+        description: >-
+         Array of logs returned in answer to the query. The logs are returned in their entirety and parsed.
+         
+         
+         If the logs are no longer retained in the database, the request will return empty. You can check your account's log retention policy in your [log monitoring account](https://app.logz.io/#/dashboard/settings/usage-and-billing).
+        items:
+          type: object
+          example: {Array of logs}
+          #additionalProperties:
+          #  type: object
+      pagination:
+        $ref: '#/definitions/Pagination'
+  PagedSearchResponseTriggeredAlert:
+    type: object
+    properties:
+      total:
+        type: integer
+        format: int32
+        description: The total number of events returned by the query. The total entities found after filtering and sorting. This number is fixed and not affected by pagination.
+        example: 500
+      results:
+        type: array
+        items:
+          $ref: '#/definitions/TriggeredAlert'
+      pagination:
+        $ref: '#/definitions/Pagination'
+  TriggeredAlert:
+    type: object
+    properties:
+      alertId:
+        type: integer
+        format: int32
+        description: Unique identifier of the security rule in Logz.io Cloud SIEM. Equivalent to the log field `logzio-alert-definition-id`
+        example: 453345
+      name:
+        type: string
+        description: Name of the security rule in Logz.io Cloud SIEM
+        example: AWS EC2 - Brute force SSH login attempts
+      description:
+        type: string
+        description: Typically an explanation of the security rule's logic and suggested next steps
+        example: Suggested next steps...
+      alertSummary:
+        type: string
+        description: Equivalent to the `condition` field in the rule
+        example: Alert if query '*' results GREATER_THAN_OR_EQUALS 5.00 in 10 minutes. Count on Group By '[userIdentity.userName, sourceIPAddress]'
```

Comment:
i'm really not sure how critical this is, and if anyone will notice, but the "group by" described here does not match the groupBy returned in below example . perhaps should be the same just for consistency? :)